src/model/kaniRunner.ts

```diff
@@ -24,6 +28,69 @@ interface CommandOutput {
 	error: any;
 }
 
+/**
+ * Get the system resolved path to the cargo-kani command. Tries to get the installed version first and if that fails, picks up the dev version automatically.
+ *
+ * @param binaryName - Full sanitized command created by kaniCommandCreate module
+ * @returns the path for the binary cargo-kani (either the installed binary or the development one)
+ */
+export async function getBinaryAbsolutePath(binaryName: string): Promise<string> {
+	try {
+		// Try using 'which' command
+		const output = await getKaniPath(binaryName);
+		if (output) {
+			return output.trim();
+		}
+	} catch (error) {
+		// Ignore 'which' command error
+	}
+
+	try {
+		// Try using glob pattern to find the binary
+		const matches = await globAsync(`kani/scripts/cargo-kani`, { absolute: true });
```

Comment:
I'm not sure how VSCode resolves this relative path.

For this to work, does the user need to have a `kani` folder in the current working directory?

To be honest, I'm not sure why this is better than setting PATH? PATH can be set to wherever the user has built kani, while this code seems to be hardcoding an assumption about where the user has checked out kani.

Reply:
Users can still set PATH. `Which cargo-kani` will pick up the PATH variable anyway. This allows us to invoke kani as a binary for the unit tests. I thought I can add them as a follow up, but if it makes things more clear, I can add a few unit tests using this piece of code.